scripts/download_wheels.sh

```diff
@@ -11,6 +11,7 @@
 # one by one to the dist/ directory where they would be generated.
 
 VERSION=$1
+mkdir -p $(dirname -- $0)/../dist
```

Comment:
Sorry for the late review. Small comment, but seems like you could just do `mkdir -p $DIST_DIR` if you write this in the line after defining the directory, no?

Reply:
You're right, I think.

Will make a note to do this (hopefully I don't forget).